data/_models/shared/Resources.csv

```diff
@@ -20,7 +20,7 @@ Resources,,external identifiers,refback,,,,External identifiers,,resource,,,Exte
 Resources,,start year,int,,,,,,,,dcat:startDate,"The year when the resource started, e.g. when data or samples started to be collected or extracted, or when the resource was established","DataCatalogueFlat,EMA,RWEStaging,CohortsStaging,UMCGCohortsStaging,NetworksStaging"
 Resources,,end year,int,,,,,,,,dcat:endDate,"The year the resource ceased, e.g. when data or sample collection was stopped","DataCatalogueFlat,EMA,RWEStaging,CohortsStaging,UMCGCohortsStaging,NetworksStaging"
 Resources,,time span description,text,,,,,,,,,Description of time span,"DataCatalogueFlat,RWEStaging"
-Resources,,contact email,,,,,,,,,dcat:contactPoint,Contact e-mail address ,"DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging"
```

Comment:
dito

Reply:
dcat:contactPoint is structured in vcard:Kind, which in HRICore and DCAT-AP3.0 needs a name and email address. So I removed this annotation, but kept the one on personsInvolved.

Reply:
so we want to drop 'contact email' and instead use the contacts?